stable/mongodb/README.md

```diff
@@ -115,6 +115,12 @@ The following table lists the configurable parameters of the MongoDB chart and t
 | `securityContext.fsGroup`                          | Group ID for the container                                                                                                                                | `1001`                                                   |
 | `securityContext.runAsUser`                        | User ID for the container                                                                                                                                 | `1001`                                                   |
 | `schedulerName`                                    | Name of the k8s scheduler (other than default)                                                                                                            | `nil`                                                    |
+| `sidecars`                                         | Add additional containers to pod                                                                                                                          | `[]`                                                     |
+| `extraVolumes`                                     | Add additional volumes to deployment                                                                                                                      | `[]`                                                     |
+| `extraVolumeMounts`                                | Add additional volumes mounts to pod                                                                                                                      | `[]`                                                     |
+| `sidecarsArbiter`                                  | Add additional containers to arbiter pod                                                                                                                  | `[]`                                                     |
+| `extraVolumesArbiter`                              | Add additional volumes to arbiter deployment                                                                                                              | `[]`                                                     |
+| `extraVolumeMountsArbiter`                         | Add additional volumes mounts to arbiter pod                                                                                                              | `[]`                                                     |
```

Comment:
What's the reasoning behind differentiating between primary/secondaries and the arbiter? What's the use case?

Reply:
@alemorcuq I mimic the `resourcesArbiter` or `affinityArbiter` fot the naming.
In case of using side cars for logging, you may want to store the logs into a different ES index for example in case of the arbiter, so you may need different volumes.

Reply:
But would you like the same for example in the case of a secondary node? At the moment there are two different type of parameters: those for the primary/secondary and those for the arbiter.

Reply:
Honestly, we are using the database in standalone mode for now, but from what I read, config for primary and secondary node should be similar.

However, if you want I can change this, but I don't think this is the spirit of other parameters in the chart, which do not differ for primary or secondary